scripts/Diffraction/isis_powder/routines/focus.py

```diff
@@ -14,20 +14,24 @@
 import os
 
 
-def focus(run_number_string, instrument, perform_vanadium_norm, absorb, sample_details=None):
+def focus(run_number_string, instrument, perform_vanadium_norm, absorb, sample_details=None,
+          absorb_method="Mayers", paalman_pings_events_per_point=None):
     input_batching = instrument._get_input_batching_mode()
     if input_batching == INPUT_BATCHING.Individual:
         return _individual_run_focusing(instrument=instrument, perform_vanadium_norm=perform_vanadium_norm,
-                                        run_number=run_number_string, absorb=absorb, sample_details=sample_details)
+                                        run_number=run_number_string, absorb=absorb,
+                                        sample_details=sample_details, absorb_method=absorb_method,
+                                        paalman_pings_events_per_point=paalman_pings_events_per_point)
     elif input_batching == INPUT_BATCHING.Summed:
         return _batched_run_focusing(instrument, perform_vanadium_norm, run_number_string, absorb=absorb,
-                                     sample_details=sample_details)
+                                     sample_details=sample_details, absorb_method=absorb_method,
+                                     paalman_pings_events_per_point=paalman_pings_events_per_point)
     else:
         raise ValueError("Input batching not passed through. Please contact development team.")
 
 
-def _focus_one_ws(input_workspace, run_number, instrument, perform_vanadium_norm, absorb, sample_details,
-                  vanadium_path):
+def _focus_one_ws(input_workspace, run_number, instrument, perform_vanadium_norm, absorb,
+                  sample_details, vanadium_path, absorb_method="Mayers", paalman_pings_events_per_point=None):
```

Comment:
One of my other comments talks about changing the values in the absorb_method parameter (and perhaps the name of the parameter itself) so this comment may be overtaken by that...
If we do keep the absorb_method parameter as is, I don't think you need the default "Mayers" values in all 4 places in this unit. It's probably sufficient just to have the default on the outer focus function